src/styles/base/_base.scss

```diff
@@ -0,0 +1,21 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 10px;
+
+  @extend %font-roboto;
+}
+
+.button {
+  display: block;
+  width: 100%;
+  border: none;
+  cursor: pointer;
+  transition: all 0.5s;
```

Comment:
It's a good practice to avoid using `all` in the `transition` property because it can lead to performance issues. Instead, specify which properties you want to transition. For example, if you're transitioning the `background-color` and `color`, you should write `transition: background-color 0.5s, color 0.5s;`.